examples/image-streams/image-streams-centos7.json

```diff
@@ -690,7 +690,7 @@
               "openshift.io/display-name": "MongoDB 2.4",
               "description": "Provides a MongoDB 2.4 database on CentOS 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/2.4/README.md.",
               "iconClass": "icon-mongodb",
-              "tags": "mongodb",
+              "tags": "hidden,mongodb",
```

Comment:
I would rather just update an example to use 2.6 (it's a trivial change and 99% it will work without any issues). But it's up to you, of course.

BTW, PetSet will be renamed to StatefulSet soon. JFYI.